### ISSUE TITLE
MacOS builds need to be signed.

### ISSUE THREAD

**Issue by shayneoneill** [touilleMan/godot-python#197]
I've been trying to install this from the Asset library. The problem is Catalinas security is amazingly obstructive. It tells me first the dylib isnt authorized and this requires finding it in the security tab, authorizing it, then restarting godot. Then the process starts again, for the the libpython.so. And then again and again and again for every .so. 

After half an hour of restarting and authorizing about 20-25 .so files and still only part way through the alphabet, I've given up. I've looked around and theres no way to automate the cycle of restart Godot, error out, fix security restart godot, error out etc etc etc. Its just too hard....

The solution is to sign the files, but I dont have an apple developer account. Covid took my job, out of my price range right now :(

**Comment by matheus2740** [comment]
hey @shayneoneill I'm not on Catalina yet so I can't say a specific solution, but I suspect you'll have a much smoother time if you compile from source.
You can also try disabling gatekeeper:
`sudo spctl --master-disable `

**Comment by shayneoneill** [comment]
Yeah I got it going from source. Did take a couple of refreshes still to get the repl working. 

Not sure what to suggest for a solution in general. It'll probably need someone with an apple account to do the installing,but it now strikes me that this problems bigger than godot-python and might well be endemic to catalina and GDNative plugins in general. 

**Brainfart:** Perhaps Godot as an organization needs to look into some sort of process with the native plugins where they get submitted as a github or something with some kind of standard Continuos Integration process that lets them build all the plugins against the cert that they use for godot itself , presumably after whoever is doing their release management does some checks to make sure nobody is sneaking anything naughty past that cert. Probably along side some sort of tool that can actually do all this for the end user as well when they go to build out and release their game.

**Comment by matheus2740** [comment]
Yeah, the REPL is kinda wonky on first startup...
On the subject of certs/signing, I have no clue how to tackle this. @touilleMan any thoughts?

**Comment by touilleMan** [comment]
The support I can provide on MacOS is pretty limited given I personally don't use it (besides the fact I have zero affection for this platform made by a band of thieves that refuses to allow virtual machine or crossplatform development and [pushes racket on devs](https://mobile.twitter.com/dhh/status/1272968382329942017) )

As @shayneoneill said this is a broader problem that just Godot-Python so basically all dev wanting to ship GDNative  module on MacOS is in need of some documentation.
So I guess it would be a really good first step to improve Godot documentation on this point ;-)

From this on we could see how complicated it would be to provide signed MacOS build for Godot-Python
